tests/pypi/evaluate_markers/BUILD.bazel

```diff
@@ -0,0 +1,7 @@
+load("@bazel_skylib//rules:build_test.bzl", "build_test")
+load("@dev_pip//:requirements.bzl", "all_whl_requirements")
+
+build_test(
+    name = "all_dev_wheels",
+    targets = all_whl_requirements,
+)
```

Comment:
This is basically an integration test that the `dev_pip` works - it is using env markers and if this target fails on CI, then we know that the feature does not work.